src/rnr_score_hv/innov_netcdf.py

```diff
@@ -0,0 +1,329 @@
+"""
+Copyright 2022 NOAA
+All rights reserved.
+
+Collection of methods to facilitate file/object retrieval
+
+"""
+from collections import namedtuple
+from dataclasses import dataclass, field
+from datetime import datetime
+import netCDF4
+
+from rnr_score_hv.config_base import ConfigInterface
+from rnr_score_hv import file_utils
+
+valid_metrics = [
```

Comment:
in a follow up PR you might want to support the fact that these innov stats might come from either the atmosphere or the ocean. For ocean, salinity is obvious but it also has temperature. which is confusing because both atm. and oce. have temperature.

Reply:
Yeah, that would be easy enough to add.

Reply:
This harvester was written specifically for the innov_stats files created by the UFS-RNR, so if the workflow produces that type of file for both ocean and atm, then it would make sense to add the option here.  The UFS-RNR doesn't distinguish between ocean or atm in this file, but I could put that in the config file.  Perhaps a new config category called 'da_type'.  If it's ok with you, I'll leave that out of this particular PR.

Reply:
ok to add later.